src/argilla_llama_index/llama_index_handler.py

```diff
@@ -101,20 +56,18 @@ def __init__(
                 "Python package installed. Please install it with `pip install argilla`"
             )
 
-        # Check whether the Argilla version is compatible
         if parse(self.ARGILLA_VERSION) < parse("1.18.0"):
             raise ImportError(
                 f"The installed `argilla` version is {self.ARGILLA_VERSION} but "
                 "`ArgillaCallbackHandler` requires at least version 1.18.0. Please "
                 "upgrade `argilla` with `pip install --upgrade argilla`."
             )
-        
-        # API_URL and API_KEY
-        # Show a warning message if Argilla will assume the default values will be used
+
+        # Ensure the API URL and API key are set, or assume the default values
         if api_url is None and os.getenv("ARGILLA_API_URL") is None:
```

Comment:
I believe the Argilla SDK has internal logic for this and the other API_KEY check